spec/02-integration/04-admin_api/15-off_spec.lua

```diff
@@ -1040,11 +1041,14 @@ describe("Admin API #off with Unique Foreign #unique", function()
     assert.equal(references.data[1].note, "note")
     assert.equal(references.data[1].unique_foreign.id, foreigns.data[1].id)
 
-    local res = assert(client:get("/cache/unique_references||unique_foreign:" ..
-                                  foreigns.data[1].id))
-    local body = assert.res_status(200, res)
-    local cached_reference = cjson.decode(body)
+    local key = "unique_references\\|\\|unique_foreign:" .. foreigns.data[1].id
+    local handle = io.popen("resty --main-conf \"lmdb_environment_path " ..
```

Comment:
This kinda thing really makes me wish lmdb could be initialized in lua land, but I'm assuming there are good reasons why that's infeasible right now.